src/middlewared/middlewared/plugins/disk.py

```diff
@@ -1298,6 +1298,13 @@ def check_clean(self, disk):
         if not klass:
             return
 
+        # Add non-mirror swap devices
+        # e.g. when there is a single disk
+        swap_devices += [
+            i.devname.strip('.eli')
```

Comment:
`devname` can end with `e` and it'll be stripped.

`disk` plugin already has another instance of this error